packages/create-llama/helpers/devcontainer.ts

```diff
@@ -0,0 +1,44 @@
+import fs from "fs";
+import path from "path";
+import { TemplateFramework } from "./types";
+
```

Comment:
i think you can directly import the JSON like this:
```suggestion
import devcontainerJson from "../templates/devcontainer.json"
```

Reply:
but then you would make to need a copy of the object, probably not worth it